docs/writing-docs/doc-block-story.md

```diff
@@ -7,46 +7,29 @@ In Storybook Docs, stories are rendered in the `Story` block.
 
-| Option          | Description                                                                                                                                                                                                   |
-| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `inlineStories` | Configures Storybook to render stories inline. <br/> `docs: { inlineStories: false }` <br/> Read the [documentation](./docs-page.md#inline-stories-vs-iframe-stories) for inline rendering framework support. |
+| Option          | Description                                                                           |
+| --------------- | ------------------------------------------------------------------------------------- |
+| `inlineStories` | Configures Storybook to render stories inline. <br/> `docs: { inlineStories: false }` |
 
 ## Working with MDX
 
 With MDX, the `Story` block is not only a way of rendering stories, but how you define them. Internally, Storybook looks for named `Story` instances located at the top of your document, or inside a [Canvas](./doc-block-canvas.md). Below is an abridged example and table featuring all the available options.
 
-| Option       | Description                                                                                                                                                                                                                                                   |
-| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `args`       | Provide the required component inputs (e.g., props). <br/> `<Story args={{ text: 'Button' }}/>` <br/> Read the [documentation](../writing-stories/args.md) to learn more.                                                                                     |
+| Option       | Description                                                                                                                                                                                                                                                    |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `args`       | Provide the required component inputs (e.g., props). <br/> `<Story args={{ text: 'Button' }}/>` <br/> Read the [documentation](../writing-stories/args.md) to learn more.                                                                                      |
```

Comment:
Given that we're removing all docs for writing stories in MDX (with a link back to older docs)...

Wouldn't all of these annotation props (`args`, `decorators`, `loaders`, `parameters`, `play`) go away as well, because they're only defined in CSF now?

Happy to update this as part of my MDX 2 PR (coming soon!); just let me know!

Reply:
Indeed they should, but this is a different conversation altogether. One that I'm going to have with @JReinhold to get some clarification. I've mentioned this in the pr description